keda/README.md

```diff
@@ -126,6 +126,8 @@ their default values.
 | `podDisruptionBudget.operator` | object | `{}` | Capability to configure [Pod Disruption Budget] |
 | `podLabels.keda` | object | `{}` | Pod labels for KEDA operator |
 | `podSecurityContext.operator` | object | [See below](#KEDA-is-secure-by-default) | [Pod security context] of the KEDA operator pod |
+| `profiling.operator.enabled` | bool | `false` | Enable profiling for keda operator |
```

Comment:
We need to make sure that your new settings are not showing up here

Reply:
So you want them to be only under the troubleshooting section?

Reply:
yes!